blockchain/indexers/existsaddrindex.go

```diff
@@ -240,10 +240,10 @@ func (idx *ExistsAddrIndex) ConnectBlock(dbTx database.Tx, block, parent *dcruti
 			// is used for the script version.  This will ultimately need to
 			// updated to support new script versions.
 			const scriptVersion = 0
-			if txscript.IsMultisigSigScript(txIn.SignatureScript) {
-				rs := txscript.MultisigRedeemScriptFromScriptSig(
+			if stake.IsMultisigSigScript(scriptVersion, txIn.SignatureScript) {
```

Comment:
The exists address index is not consensus and therefore nothing in this file should be updated.